package.json

```diff
@@ -135,6 +135,8 @@
   "devDependencies": {
     "@apidevtools/swagger-parser": "10.1.0",
     "@babel/core": "7.18.13",
+    "@swc/core": "1.2.246",
+    "@swc/jest": "0.2.22",
```

Comment:
These little guys intentional in this PR?

Reply:
Err, no. No, they're not. I'll remove them. Good spot! 🙌🏼

Reply:
No wait, hang on! They've just been sorted! Don't know why, but I'm not against it. I'll leave it in.